### ISSUE TITLE
load time is slow

### ISSUE THREAD

**Issue by tansaku** [AgileVentures/av-ghost-theme#13]
because on heroku hobby tier ... should we:

1) pay them
2) try digital ocean
3) use jekyll?

**Comment by diraulo** [comment]
@tansaku my Argument for ghost.io, was the ease of use for non-developers. If articles are only posted by developers, then Jekyll or rather the [middleman-blog](https://github.com/middleman/middleman-blog) gem would do the trick for the blog.

**Comment by tansaku** [comment]
yeah, funny thing is that we were anticipating charles posting on it as a non-developer - but now's he's saying he might send me word docs ... :-)
